ddtrace/tracer/tracer.go

```diff
@@ -6,7 +6,6 @@
 package tracer
 
 import (
-	"context"
 	gocontext "context"
```

Comment:
Not exactly sure I understand the reason behind this change.

Reply:
Importing the `context` pkg twice under different aliases is weird. Importing it as `context` was an accident. I chose `gocontext` to avoid shadowing an existing local variable in the code I was editing for the endpoints/hotspots PR, but one reference to context crept in.

Reply:
Got it. It wasn't obvious from this PR that another file in the same package imports it differently.